.drone.yml

```diff
@@ -95,37 +84,3 @@ steps:
     branch:
       exclude:
       - main
-
----
-kind: pipeline
```

Comment:
Oh wait hang on, did you mean to trash the build notification?

Reply:
Not that we’re really looking at these at the min as far as I’m aware 😅

Reply:
I did not I have added it back